ArduCopter/Parameters.cpp

```diff
@@ -258,42 +258,42 @@ const AP_Param::Info Copter::var_info[] = {
 
     // @Param: FLTMODE1
     // @DisplayName: Flight Mode 1
-    // @Description: Flight mode when Channel 5 pwm is <= 1230
+    // @Description: Flight mode when pwm of Flightmode channel is <= 1230
```

Comment:
You could add a reference to the param name. 
```suggestion
    // @Description: Flight mode when pwm of Flightmode channel (FLTMODE_CH) is <= 1230
```

Reply:
@IamPete1 san.
I agree with this comment.
I'll do the same for the other flight mode settings.